backend/app/handlers/requests.js

```diff
@@ -1,55 +1,60 @@
-    }
-    }, 60000);
-
-  app.use(compression({ filter: (req, res) => true, level: 6, algorithms: ['br', 'gzip', 'deflate'] }));
-  app.use(cors({
-    origin: function (origin, callback) {
-      if (!origin) return callback(null, true);
-      if (origins.indexOf(origin) !== -1) {
-        callback(null, true)
-      } else {
-        console.log("Express: Blocked by CORS: " + origin);
-        callback('Not allowed by CORS')
-      }
-    }
-  }));
-  app.use(express.json());
-  app.use(helmet({
-    contentSecurityPolicy: {
-      directives: {
-        defaultSrc: ["'self'", "https://unpkg.com", "'unsafe-inline'", "https://123rf.com"],
-        scriptSrc: ["'self'", "'unsafe-inline'", "'wasm-unsafe-eval'"],
-      },
-    },
-  }));
-
-  app.use("/api/user", dependencies.userRoutes || require("../routes/userRoutes"));
-  app.use("/api/owner", dependencies.ownerRoutes || require("../routes/ownerRoutes"));
-  app.use("/api/status", dependencies.statusRoutes || require("../routes/statusRoutes"));
-  app.use("/api/app", dependencies.appRoutes || require("../routes/appRoutes"));
-  app.use("/", express.static(frontendFilesPath));
-  return app;
-}
+	const app = express();
+
+	let origins = require('../../cors.json').origins;
+	console.log(origins);
+	setInterval(() => {
+		try{
+			delete require.cache[require.resolve('../../cors.json')];
+			origins = require('../../cors.json').origins;
+		} catch(e) {
+			console.log('Express: Error reading cors.json file');
+		}
+	}, 60000);
+
+	app.use(compression({ filter: () => true, level: 6, algorithms: ['br', 'gzip', 'deflate'] }));
+	app.use(cors({
+		origin: function (origin, callback) {
+			if (!origin) return callback(null, true);
+			if (origins.indexOf(origin) !== -1) {
+				callback(null, true);
+			} else {
+				console.log('Express: Blocked by CORS: ' + origin);
+				callback('Not allowed by CORS');
+			}
+		}
+	}));
+	app.use(express.json());
+	app.use(helmet({
+		contentSecurityPolicy: {
+			directives: {
+				defaultSrc: ['\'self\'', 'https://unpkg.com', '\'unsafe-inline\'', 'https://123rf.com'],
+				scriptSrc: ['\'self\'', '\'unsafe-inline\'', '\'wasm-unsafe-eval\'', 'blob:', 'https://cdn.jsdelivr.net'],
+				imgSrc: ['\'self\'', 'https://previews.123rf.com']
+			},
+		},
+	}));
+
+	app.use('/api/user', dependencies.userRoutes || require('../routes/userRoutes'));
+	app.use('/api/owner', dependencies.ownerRoutes || require('../routes/ownerRoutes'));
+	app.use('/api/status', dependencies.statusRoutes || require('../routes/statusRoutes'));
+	app.use('/api/app', dependencies.appRoutes || require('../routes/appRoutes'));
+
+	app.use('/@:random', (req) => req.url = '', express.static(frontendFilesPath));
+	app.use('/join', (req) => req.url = '', express.static(frontendFilesPath));
+	app.use('/', express.static(frontendFilesPath));
+
+	return app;
+};
 
 module.exports = {
-    createApp
-}
\ No newline at end of file
+	createApp
+};
\ No newline at end of file
```

Comment:
Overall, the code patch looks good and well-formatted. Here are some suggestions for improvement:

1. It's a good practice to maintain the consistency of single/double quotes usage. You can choose either single or double quotes and use them consistently throughout your code.

2. In line with the above suggestion, make sure you update all instances of quotes in the code when changing them, including quotes used in objects like `directives`.

3. On line 29 of the code patch, the `filter` function inside the `compression` middleware currently does not apply any filter to the responses. Consider refining the function to selectively compress specific response types only, to further optimize the application's performance.

4.  On lines 48-51, consider defining error-handling middleware to catch and handle CORS rejection errors in the application instead of simply printing an error message to the console. 

5. On lines 64-66, it might be beneficial to remove `require.cache[require.resolve('../../cors.json')]` or reduce its interval time if loading and reloading CORS configurations excessively affects your application's performance or I/O operations.

6. Consider adding more comments that help explain the purpose and functions of each part of the code. This will enhance readability and understanding, making the code more maintainable over time.

7. Lastly, add a newline at the end of the file to adhere to common practice and avoid issues with version control tools.